source/product-overview/common-esr-support-upgrade.md

```diff
@@ -1,6 +1,6 @@
 <!-- Snippet include; not intended to be a standalone page -->
 
 - Support for Mattermost Server v10.5 [Extended Support Release](https://docs.mattermost.com/product-overview/release-policy.html#extended-support-releases) is coming to the end of its life cycle on November 15, 2025. Upgrading to Mattermost Server v10.11 or later is recommended.
-- Upgrading from ESR-to-ESR (``major`` -> ``major_next``) is fully supported and tested. However, upgrading from ESR-to-ESR (``major`` to ``major+2``) is supported, but not tested. If you plan to upgrade across multiple releases, we strongly recommend upgrading from an ESR to another ESR. For example, if you're upgrading from the v8.1 ESR, upgrade to the [v9.5 ESR](https://docs.mattermost.com/about/mattermost-v9-changelog.html#release-v9-5-extended-support-release) or the [v9.11 ESR](https://docs.mattermost.com/about/mattermost-v9-changelog.html#release-v9-11-extended-support-release).
+- Upgrading from ESR-to-ESR (``major`` -> ``major_next``) is fully supported and tested. However, upgrading from ESR-to-ESR (``major`` to ``major+2``) is supported, but not tested. If you plan to upgrade across multiple releases, we strongly recommend upgrading from an ESR to another ESR. For example, if you're upgrading from the v8.1 ESR, upgrade to the [v9.5 ESR](https://docs.mattermost.com/product-overview/unsupported-legacy-releases.html#release-v9-5-extended-support-release) or the [v9.11 ESR](https://docs.mattermost.com/product-overview/unsupported-legacy-releases.html#release-v9-11-extended-support-release).
```

Comment:
I understand this PR only addresses the version numbers in this parapgraph but we should probably update this to a) be more clear, I find the wording confusing and b) also make sure we don't link to the "unsupported legacy releases" changelog when we explain how to upgrade between ESR releases.